source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/universal_robots.py

```diff
@@ -51,4 +52,57 @@
         ),
     },
 )
-"""Configuration of UR-10 arm using implicit actuator models."""
+
+"""Configuration of UR5e arm using implicit actuator models."""
+
+UR5E_CFG = ArticulationCfg(
+    spawn=sim_utils.UsdFileCfg(
+        usd_path="omniverse://localhost/NVIDIA/Assets/Isaac/4.2/Isaac/Robots/UniversalRobots/ur5e/ur5e.usd",
```

Comment:
I will ask this many many times I think, this path is there because you have nucleus or is this usd available publicly by downloading isaac sim?

Reply:
It is publicly downloadable from the isaac sim folder in Isaac Nucleus and not from our own AICA Assets.